vulcanai/datasets/tabular_data_utils.py

```diff
@@ -0,0 +1,394 @@
+# -*- coding: utf-8 -*-
+"""
+This file defines a set of functions useful for working with pandas
+dataframes full of tabular data
+"""
+
+import numpy as np
+import pandas as pd
+import logging
+from itertools import groupby
+from sklearn import preprocessing
+
+logger = logging.getLogger(__name__)
+
+# TODO: split function
+
+
+# TODO: use kwargs
+def create_label_encoding(df, column_name, ordered_values):
```

Comment:
is there any functionality that we are losing with these split data util functions as compared to the class-based implementation?

Reply:
also there are a few functions that got removed in this transition around saving dataframes? such as listing features, listing columns, etc

Reply:
No, not if the user users another split function on their dataframe or on their tabular data. I'd like to move away from us performing splits in any way but very deliberately and manually - generate a test/train/validation set BEFORE anything else happens in the analysis.

Reply:
yes, I got rid of all the features that were very simple wrappers around simple pandas functions

Reply:
ok yeah that makes sense. Trimming the unnecessary duplicated fat